package on2015_07.on2015_07_23_101_Hack_Jul_2015.Special_Arrays_of_1_and_2;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("lib/test/on2015_07/on2015_07_23_101_Hack_Jul_2015/Special_Arrays_of_1_and_2/Special Arrays of 1 and 2.task"))
			Assert.fail();
	}
}
